app/sprinkles/account/templates/pages/set-password.html.twig

```diff
@@ -17,8 +17,8 @@
     <!-- /.login-logo -->
 
     <div class="login-box-body login-form">
-        <p class="login-box-msg"><strong>Create Password</strong></p>
-        <p class="login-box-msg">Welcome to {{site.title}}!  Please create a password for your new account.</p>
+        <p class="login-box-msg"><strong>{{translate("PASSWORD.CREATE")}}</strong></p>
+        <p class="login-box-msg">{{translate("WELCOME_TO", {'title': site.title})}} {{translate("PASSWORD.CREATE.PAGE")}}</p>
```

Comment:
Would this be a good spot to use [compound i18n keys](https://learn.userfrosting.com/building-pages/i18n#Theplaceholder)?  `PASSWORD.CREATE.PAGE` could reference the `WELCOME_TO` key.